server/@types/dto/index.d.ts

```diff
@@ -73,4 +73,10 @@ declare module 'dto' {
     code: string
     areaCode?: string
   }
+
+  export interface AlnScreenerDto {
+    prisonNumber: string
+    prisonId: string
+    screenerDate?: Date
```

Comment:
We will add extra fields as we progress through the ALN journey stories - at the moment I am only interested in the "screener date" screen, so only need these fields